scripts/precommit_hook.js

```diff
@@ -6,5 +6,21 @@
  * Side Public License, v 1.
  */
 
+var Fs = require('fs');
+var Path = require('path');
+var Cp = require('child_process');
+
+var REPO_ROOT = Path.resolve(__dirname, '..');
+if (Fs.existsSync(Path.resolve(REPO_ROOT, '.git/MERGE_HEAD'))) {
+  process.stdout.write(
+    'Bootstrapping before running pre-commit hook, now that merge is applied...\n'
+  );
+  Cp.execFileSync('yarn', ['kbn', 'bootstrap'], {
```

Comment:
With the changes to TsProject loading logic to automatically rebuild the tsconfig-paths.json cache, is this here just a convenience for getting a more correct state after resolving a merge, or it is still a necessity ?

Reply:
I'm not certain it's still useful, maybe we should just remove it... Removing.